From b138ff0dff8df4800c4cad9372bc9e6cdbd0408d Mon Sep 17 00:00:00 2001
From: Kristen Carlson Accardi <kristen@linux.intel.com>
Date: Tue, 21 Jun 2016 05:33:03 +0000
Subject: [PATCH 0089/2508] intel_pstate: add support for BXT-P

commit 4b9fdfa246ed892c48951f24fd4bec234920c402 from
https://github.com/01org/linux-apollolake-i

BXT-P cpuid added to whitelist.  BXT-P needs to use core_params
because the MSRs for BYT/CHT don't work on BXT.  Tuning for
BXT is TBD.

Signed-off-by: Kristen Carlson Accardi <kristen@linux.intel.com>
Signed-off-by: Yong, Jonathan <jonathan.yong@intel.com>
Signed-off-by: Bruce Ashfield <bruce.ashfield@windriver.com>
---
 drivers/cpufreq/intel_pstate.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/drivers/cpufreq/intel_pstate.c b/drivers/cpufreq/intel_pstate.c
index 6a308e2..a197ad9 100644
--- a/drivers/cpufreq/intel_pstate.c
+++ b/drivers/cpufreq/intel_pstate.c
@@ -924,6 +924,7 @@ static const struct x86_cpu_id intel_pstate_cpu_ids[] = {
 	ICPU(0x4c, byt_params),
 	ICPU(0x4e, core_params),
 	ICPU(0x4f, core_params),
+	ICPU(0x5c, core_params),
 	ICPU(0x5e, core_params),
 	ICPU(0x56, core_params),
 	ICPU(0x57, knl_params),
-- 
1.7.5.4

